proto/rill/runtime/v1/api.proto

```diff
@@ -347,23 +343,11 @@ message DeleteInstanceResponse {}
 // See message Instance for field descriptions.
 message EditInstanceRequest {
   string instance_id = 1 [(validate.rules).string = {pattern: "^[_\\-a-zA-Z0-9]+$"}];
-  optional string olap_driver = 2 [(validate.rules).string = {
-    in: [
-      "duckdb",
-      "druid"
-    ]
-  }];
-  optional string olap_dsn = 3;
-  optional string repo_driver = 4 [(validate.rules).string = {
-    in: [
-      "file",
-      "metastore",
-      "github"
-    ]
-  }];
-  optional string repo_dsn = 5;
+  optional string olap_driver = 2;
+  optional string repo_driver = 4;
   optional bool embed_catalog = 6;
   optional int64 ingestion_limit_bytes = 8;
+  repeated Connector connectors = 9;
```

Comment:
For `EditInstance`, how will it know if this field is `optional` or nil? We might need a separate API like for variables